README.md

```diff
@@ -1,4 +1,56 @@
-# ActiveModelSerializers
+# ActiveModelSerializers (Deprecated)
```

Comment:
should link down to https://github.com/rails-api/active_model_serializers#status-of-ams